velox/exec/Window.cpp

```diff
@@ -475,6 +475,9 @@ void Window::callApplyLoop(
       break;
     }
   }
+
+  // callApplyLoop returns the number of rows processed by it.
```

Comment:
It is strange for the function to talk about itself in 3rd person.

// Return the number of processed rows.